pkg/plugins/queue/task.go

```diff
@@ -12,13 +12,12 @@
 // See the License for the specific language governing permissions and
```

Comment:
Would recommend moving these types into the ```plugin.go``` file.

Reply:
see above